mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/utbetaling/UtbetalingServiceTest.kt

```diff
@@ -66,21 +81,102 @@ class UtbetalingServiceTest : FunSpec({
 
-    coEvery { kontoregisterOrganisasjonClient.getKontonummerForOrganisasjon(Organisasjonsnummer("976663934")) } returns Either.Right(
-        KontonummerResponse(
-            mottaker = "976663934",
-            kontonr = "12345678901",
-        ),
-    )
+    beforeTest {
+        coEvery { kontoregisterOrganisasjonClient.getKontonummerForOrganisasjon(Organisasjonsnummer("123456789")) } returns Either.Right(
+            KontonummerResponse(
+                mottaker = "123456789",
+                kontonr = "12345678901",
+            ),
+        )
+
+        coEvery { kontoregisterOrganisasjonClient.getKontonummerForOrganisasjon(Organisasjonsnummer("976663934")) } returns Either.Right(
+            KontonummerResponse(
+                mottaker = "976663934",
+                kontonr = "12345678901",
+            ),
+        )
+
+        coEvery {
+            pdl.hentPersonOgGeografiskTilknytningBolk(
+                any(),
+                any(),
+            )
+        } returns Either.Right(
+            mapOf(
+                PdlIdent("12345678910") to Pair(
+                    HentPersonBolkResponse.Person(
+                        navn = nonEmptyListOf(
+                            PdlNavn(fornavn = "Ola", etternavn = "Normann"),
+                        ),
+                        adressebeskyttelse = listOf(
+                            HentPersonBolkResponse.Adressebeskyttelse(gradering = PdlGradering.UGRADERT),
+                        ),
+                        foedselsdato = nonEmptyListOf(
+                            HentPersonBolkResponse.Foedselsdato(foedselsaar = 1980, foedselsdato = null),
+                        ),
+                    ),
+                    GeografiskTilknytningResponse.GtBydel(
+                        value = "030102",
+                    ),
+                ),
+                PdlIdent("99887766554") to Pair(
+                    HentPersonBolkResponse.Person(
+                        navn = nonEmptyListOf(
+                            PdlNavn(fornavn = "Kari", etternavn = "Normann"),
+                        ),
+                        adressebeskyttelse = listOf(
+                            HentPersonBolkResponse.Adressebeskyttelse(gradering = PdlGradering.STRENGT_FORTROLIG),
+                        ),
+                        foedselsdato = nonEmptyListOf(
+                            HentPersonBolkResponse.Foedselsdato(foedselsaar = 1980, foedselsdato = null),
+                        ),
+                    ),
+                    GeografiskTilknytningResponse.GtBydel(
+                        value = "030102",
+                    ),
+                ),
+            ),
+        )
+
+        coEvery { norg2Client.hentEnhetByGeografiskOmraade(any()) } returns Norg2EnhetDto(
+            enhetId = 1,
+            navn = "Nav Gjovik",
+            enhetNr = NavEnhetNummer("0502"),
+            status = Norg2EnhetStatus.AKTIV,
+            type = Norg2Type.LOKAL,
+        ).right()
+    }
```

Comment:
Er all denne test-setup'en kun relevant for én test?

I så fall kunne det kanskje vært ryddig å legge dette inn under en egen `context("deltakere for kostnadsfordeling")` slik at det blir tydligere at dette ikke er ting som kreves for alle andre testene i denne modulen.

Reply:
Evt. øverst i testen under (hvis dette er eneste test-case som krever denne mockingen).